tests/gui/models/settings.spec.js

```diff
@@ -33,10 +33,10 @@ describe('Browser: settings', function() {
 
     describe('.set()', function() {
 
-      it('should throw if the key is not supported', function() {
```

Comment:
Given that 'not supported keys' are no longer a thing, maybe it makes sense for `SUPPORTED_KEYS` to be renamed to something like `DEFAULT_KEYS`?